pkg/skaffold/configlocations/types.go

```diff
@@ -0,0 +1,308 @@
+/*
```

Comment:
could this package be put until `pkg/skaffold/config/packagename` or `pkg/skaffold/parser/packagename`? Might make it easier to find when looking for related code

Reply:
moved `/pkg/skaffold/configlocations` under `parser` -> `pkg/skaffold/parser/configlocations`